### ISSUE TITLE
Bug: unable `yarn dev`

### ISSUE THREAD

**Issue by NWYLZW** [grafana/plugin-tools#200]
<img width="1145" alt="image" src="https://user-images.githubusercontent.com/51358815/217427940-117bbdb0-16da-4e43-a674-f718a026e7a6.png">

Now, I just use `create-plugin` command and `yarn dev` in plugin directory.

<img width="657" alt="image" src="https://user-images.githubusercontent.com/51358815/217428522-fdaf92c7-1efd-4188-9f1c-cc36a14f1ae2.png">

**Comment by academo** [comment]
Hi @NWYLZW it is not clear from your issue what is the problem you are facing.

I did a test creating a panel plugin with create-plugin and running `yarn install` and later `yarn dev` which worked as expected.

From your screenshot it looks like you are missing some configuration files that are inside the `.config` folder that is generated by create-plugin.

Are you sure your plugin was generated with create-plugin? Maybe you deleted some files without noticing?

Please let me know if this is still an issue

**Comment by NWYLZW** [comment]
Yes, I am using the project created by `create-plugin` cli, and `.config/tsconfig.json` also exists in my directory, but he does not seem to read the corresponding configuration, I will try to reproduce the problem a little later.

I was able to solve this problem by moving the files in the `.config/tsconfig` directory to the project root directory.
But I think cli should also need to fix this problem.

**Comment by academo** [comment]
@NWYLZW create-plugin generates the `.config` file in the root of the project. 

E.g.: you generate a plugin "grafana-test-panel"

it will generate an structure like this:
```
.
.
├── CHANGELOG.md
├── .config
│   ├── Dockerfile
│   ├── .eslintrc
│   ├── jest
│   │   ├── mocks
│   │   │   └── react-inlinesvg.tsx
│   │   └── utils.js
│   ├── jest.config.js
│   ├── jest-setup.js
│   ├── .prettierrc.js
│   ├── README.md
│   ├── tsconfig.json
│   ├── types
│   │   └── custom.d.ts
│   └── webpack
│       ├── constants.ts
│       ├── utils.ts
│       └── webpack.config.ts
├── cypress
│   └── integration
│       └── 01-smoke.spec.ts
├── docker-compose.yaml
├── .eslintrc
├── .github
│   └── workflows
│       └── compatible.yml
├── .gitignore
├── jest.config.js
├── jest-setup.js
├── LICENSE
├── .nvmrc
├── package.json
├── .prettierrc.js
├── README.md
├── src
│   ├── components
│   │   └── SimplePanel.tsx
│   ├── img
│   │   └── logo.svg
│   ├── module.test.ts
│   ├── module.ts
│   ├── plugin.json
│   ├── README.md
│   └── types.ts
├── tsconfig.json
77 directories, 117 files

```
